setup/convert.lua

```diff
@@ -135,6 +135,26 @@ local function ConvertESX()
 end
 
 local function ConvertQB()
+
+	local function convertQbItems(items)
+		local converted = {}
+		local slot = 0
+		for _, v in pairs(items) do
+			if Items(v?.name) and v.amount then
```

Comment:
I kept the v.amount check from looking at the other examples in this file, but I'm not sure if it's possible for a qb item to have no amount.